cime_config/acme/machines/config_batch.xml

```diff
@@ -210,23 +210,32 @@
      </directives>
    </batch_system>
 
-   <batch_jobs>
-     <job name="case.run">
-       <template>template.case.run</template>
-       <task_count>default</task_count>
-     </job>
-     <job name="case.test">
-       <template>python.case.test</template>
-       <task_count>default</task_count>
-     </job>
-     <job name="case.st_archive">
-       <template>template.st_archive</template>
-       <task_count>1</task_count>
-     </job>
-     <job name="case.lt_archive">
-       <template>template.lt_archive</template>
-       <task_count>1</task_count>
-     </job>
-   </batch_jobs>
+  <batch_jobs>
```

Comment:
This puts all of the interdependancy information for template files in config_batch.xml and removes it from code.  If you want to add an additional template you should be able to it by only adding code here.